src/tests/end-to-end/tests/details-view/tabstop.test.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { launchBrowser } from '../../common/browser-factory';
+import { fastPassSelectors } from '../../common/element-identifiers/common-selectors';
+import { popupPageElementIdentifiers } from '../../common/element-identifiers/popup-page-element-identifiers';
+import { Page } from '../../common/page-controllers/page';
+import { TargetPage } from '../../common/page-controllers/target-page';
+import { Browser } from './../../common/browser';
+import { DEFAULT_PAGE_ELEMENT_WAIT_TIMEOUT_MS } from './../../common/timeouts';
+
+describe('Tabstop tests', () => {
+    describe('tabstop from fastpass', () => {
+        let browser: Browser;
+        let targetPage: TargetPage;
+        let fastPassPage: Page;
+
+        beforeAll(async () => {
+            browser = await launchBrowser({ suppressFirstTimeDialog: true });
+            targetPage = await browser.newTargetPage();
+            fastPassPage = await gotoFastPass(browser, targetPage.tabId);
```

Comment:
should we be using `DetailsViewPage` (`src\tests\end-to-end\common\page-controllers\details-view-page.ts`) which has `switchToFastPass()`???

Reply:
Will go through the new implementation and update this!